ownCloud/Client/ClientQueryViewController.swift

```diff
@@ -746,40 +603,18 @@ extension ClientQueryViewController: UISearchResultsUpdating {
 // MARK: - ClientItemCell Delegate
 extension ClientQueryViewController: ClientItemCellDelegate {
 	func moreButtonTapped(cell: ClientItemCell) {
-		if let item = cell.item {
-
-			let tableViewController = MoreStaticTableViewController(style: .grouped)
-			let header = MoreViewHeader(for: item, with: core)
-			let moreViewController = MoreViewController(item: item, core: core, header: header, viewController: tableViewController)
-
-			let title = NSAttributedString(string: "Actions", attributes: [NSAttributedStringKey.font: UIFont.systemFont(ofSize: 20, weight: .heavy)])
-
-			let deleteRow: StaticTableViewRow = StaticTableViewRow(buttonWithAction: { (_, _) in
-				moreViewController.dismiss(animated: true, completion: {
-					self.delete(item)
-				})
-			}, title: "Delete".localized, style: .destructive)
-
-			let renameRow: StaticTableViewRow = StaticTableViewRow(buttonWithAction: { [weak self] (_, _) in
-				moreViewController.dismiss(animated: true, completion: {
-					self?.rename(item)
-				})
-			}, title: "Rename".localized, style: .plainNonOpaque)
+		guard let item = cell.item else {
+			return
+		}
 
-			let moveRow: StaticTableViewRow = StaticTableViewRow(buttonWithAction: { [weak self] (_, _) in
-				moreViewController.dismiss(animated: true, completion: {
-					self?.move(item)
-				})
-				}, title: "Move".localized, style: .plainNonOpaque)
+		let actionsLocation = OCExtensionLocation(ofType: OCExtensionType.action, identifier: nil)
```

Comment:
Please add the location identifier here:
```suggestion
		let actionsLocation = OCExtensionLocation(ofType: .action, identifier: .moreItem)
```